packages/babel-helper-modules/src/index.js

```diff
@@ -209,15 +217,22 @@ const namespaceReexport = template(`
     });
   });
 `);
+const buildNameListCheck = template(`
+  if (Object.prototype.hasOwnProperty.call(EXPORTS_LIST, key)) return;
+`);
```

Comment:
Just curious, why not inline this code inside `namespaceReexport`? At first glance I thought `VERIFY_NAME_LIST` was a statement without side effect :p

Reply:
I split it off like this because when there isn't a list, like if there were no named imports, we can leave the whole `if` block out. If it's included in `namespaceReexport` then it'll always be there.